### PR TITLE
Add issue template for bug and feature

### PR DESCRIPTION
**Pull request by abhishekmishragithub**
## Description
DEV-1159

Issue #1 

What functionality does this add/problem does this fix?

- Add issue template for bug and feature using [GitHub template builder](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/about-issue-and-pull-request-templates)

This is a 🐁 change.

## Checklist
- [x] Set an informative, carefully chosen **title**
- [x] Applied *labels* to the PR
- [ ] Updated README.md and inline documentation as appropriate
- [x] Followed [Coding standards](/docs/coding-standard.md#coding-standard)
- [ ] Added TODO comments about the work not shipped in this PR
- [ ] Documented bugs/issues you had to work around

### CODE REVIEW

**Comment by tchklovski** [comment]
Task linked: [DEV-1159 Create Issue template in .github repo](https://app.clickup.com/t/8640607/DEV-1159)

**Review by tchklovski — APPROVED** [review]
nice, but should we have version and source in the comment, like for PR template?

**Comment by abhishekmishragithub** [comment]
> nice, but should we have version and source in the comment, like for PR template?

Yes. will update 👨🏼‍✈️

**Comment by abhishekmishragithub** [comment]
@tchklovski please review and merge :shipit:

**Comment by tchklovski** [comment]
@abhishekmishragithub either good to merge OR you could make text that is instructions be html comments.
In my experience, people leave such prompts in place, which they don't serve the reader.
so eg:
`A clear and concise description of what the bug is.` ⇒

`<!--A clear and concise description of what the bug is.-->`

**Comment by abhishekmishragithub** [comment]
> @abhishekmishragithub either good to merge OR you could make text that is instructions be html comments.
> In my experience, people leave such prompts in place, which they don't serve the reader.
> so eg:
> `A clear and concise description of what the bug is.` ⇒
> 
> `<!--A clear and concise description of what the bug is.-->`

Fixed it 🐥 , please review.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+ <!-- If applicable add following details
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
+ -->
+
+**Additional context:**
+<!-- Add any other context about the problem here. -->
+
+<!-- version 0.1.0 -->
+<!-- based on https://github.com/predictionmachine/.github/blob/main/issue-template/.github/ISSUE_TEMPLATE/bug_report.md -->
```
Comment (tchklovski) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
<!-- based on https://github.com/predictionmachine/.github/blob/main/.github/ISSUE_TEMPLATE/bug_report.md --> 
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+<!-- A clear and concise description of what you want to happen. -->
+
+**Describe alternatives you've considered**
+<!-- A clear and concise description of any alternative solutions or features you've considered. -->
+
+**Additional context**
+<!-- Add any other context or screenshots about the feature request here. -->
+
+<!-- version 0.1.0 -->
+<!-- based on https://github.com/predictionmachine/.github/blob/main/issue-template/.github/ISSUE_TEMPLATE/feature_request.md -->
```
Comment (tchklovski) [inline .github/ISSUE_TEMPLATE/feature_request.md]:
```suggestion
<!-- based on https://github.com/predictionmachine/.github/blob/main/.github/ISSUE_TEMPLATE/feature_request.md --> 
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+-->
+
+
+**Expected behavior:**
+<!-- A clear and concise description of what you expected to happen. -->
+
+**Screenshots:**
+<!-- If applicable, add screenshots to help explain your problem. -->
+
+**Desktop (please complete the following information):**
```
Comment (tchklovski) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
## Context
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+ - OS: <!-- [e.g. macOS] -->
+ <!-- If applicable add following details
+ - Browser [e.g. chrome, safari]
```
Comment (tchklovski) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
 - Browser: [e.g. chrome, safari]
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+ - Version [e.g. 22]
```
Comment (tchklovski) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
 - Version: [e.g. 22]
```

**Review by tchklovski — APPROVED** [review]
lgtm after a fix of the reference path

**Pull request merged**